### PR TITLE
Create nominal nofield configuration

### PR DESCRIPTION
**Pull request by brownd1978**

### CODE REVIEW

File: JobConfig/cosmic/NominalNoFieldCRY.fcl [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#include "Production/JobConfig/cosmic/NoField.fcl"
+source.module_type: EmptyEvent
+physics.producers.generate : @local::Cosmic.generateCRY
+outputs.PrimaryOutput.fileName : "dts.owner.CosmicCORSIKANominalNoField.version.sequencer.art"
```
Comment (soleti) [inline JobConfig/cosmic/NominalNoFieldCRY.fcl:4]: The output should be CRY and not CORSIKA here

**Review by soleti — CHANGES_REQUESTED** [review]

File: JobConfig/common/geom_nofield.txt [hunk 2]
```diff
@@ -1,13 +1,4 @@
...
-  "BFieldMaps/Mau13/DS-nofield.txt",
-  "BFieldMaps/Mau13/PSMap.header",
-  "BFieldMaps/Mau13/TSuMap.header",
-  "BFieldMaps/Mau13/TSdMap.header",
-  "BFieldMaps/Mau13/PStoDumpAreaMap.header",
-  "BFieldMaps/Mau13/ProtonDumpAreaMap.header",
-  "BFieldMaps/Mau13/DSExtension.header"
-};
+#include "Offline/Mu2eG4/geom/geom_common.txt"
+#include "Offline/Mu2eG4/geom/bfgeom_no_field.txt"
```
Comment (kutschke) [inline JobConfig/common/geom_nofield.txt:3]: Once Offline PR #685 is merged, you will be able to replace the two lines with just
#include "Offline/Mu2eG4/geom/geom_common_no_field.txt"

File: JobConfig/cosmic/NominalNoFieldCRY.fcl (hunk 1 above)
Comment (brownd1978) [reply JobConfig/cosmic/NominalNoFieldCRY.fcl]: thanks, fixed.

File: JobConfig/common/geom_nofield.txt (hunk 2 above)
Comment (brownd1978) [reply JobConfig/common/geom_nofield.txt]: good suggestion, done.

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Review by kutschke — APPROVED** [review]
It was a good idea to remove the now superfluous JobConfig/common/geom_nofield.txt

**Comment by kutschke** [comment]
@soleti A gentle reminder that this is in your queue.

**Review by soleti — APPROVED** [review]

**Pull request merged**